tests/test_dashboard_quadrants.py

```
@@ -51,3 +60,69 @@ async def test_jobs_preview_displays_jobs():
+        "Watcher": {
+            "check_interval": 30,
+            "enabled": True,
+            "very_long_value_that_should_be_truncated": "x" * 30,
+        },
+        "WebSocket": {
+            "enabled": False,
+        },
+    }
+
+    app = ConfigPreviewTestApp(config)
+    async with app.run_test() as pilot:
```

Ruff (0.14.14)

Local variable `pilot` is assigned to but never used

Remove assignment to unused variable `pilot`

(F841)